Dockerfile

```diff
@@ -0,0 +1,10 @@
+## relay-dapp dockerfile (This file is called Dockerfile) on server side. ##
+FROM node:12-alpine AS compiler
+RUN apk add --no-cache build-base git bash
+WORKDIR /usr/src/app
+COPY package*.json ./
+COPY .env ./.env
```

Comment:
Why do we need to specify the destination filename?
I mean `COPY .env ./` is enough, isn't it?

Reply:
Changed thanks